docs/calico-with-docker/docker-network-plugin/vagrant-ubuntu/Vagrantfile

```diff
@@ -70,7 +70,7 @@ Vagrant.configure(2) do |config|
       host.vm.provision :shell, inline: %Q|sudo sh -c 'echo "DOCKER_OPTS=\"--cluster-store=etcd://#{primary_ip}:2379\"" > /etc/default/docker'|
 
       # Restart docker.
-      host.vm.provision :shell, inline: "sudo start docker"
+      host.vm.provision :shell, inline: "sudo start docker || true"
```

Comment:
What is the fail case you were hitting here?  I'm just wondering why the || true is necessary?